src/test/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionControllerTest.java

```diff
@@ -111,12 +129,17 @@ void createObjectionTest() {
 
     @Test
     void createObjectionEligibilityErrorTest() {
+        ObjectionCreate objectionCreate = new ObjectionCreate();
+        objectionCreate.setFullName(FULL_NAME);
+        objectionCreate.setShareIdentity(false);
         Objection objection = new Objection();
         objection.setId(OBJECTION_ID);
         objection.setStatus(ObjectionStatus.INELIGIBLE_COMPANY_STRUCK_OFF);
-        when(objectionService.createObjection(REQUEST_ID, COMPANY_NUMBER, AUTH_ID, AUTH_USER)).thenReturn(objection);
+        when(objectionService.createObjection(eq(REQUEST_ID), eq(COMPANY_NUMBER),
+                eq(AUTH_ID), eq(AUTH_USER), any())).thenReturn(objection);
```

Comment:
same here, you can replace any() with the expected objectionCreate

Reply:
Updated